res/controllers/midi-components-0.0.js

```diff
@@ -332,7 +332,7 @@
         },
         outputColor: function (colorCode) {
             if (this.colors !== undefined) {
-                var nearestColor = color.nearestColorMidiCode(colorCode, this.colors);
+                var nearestColor = this.colors.getNearestColor(colorCode);
```

Comment:
How about calling it `this.colorMapper` instead of `this.colors`?

Reply:
We could also rename the JS class instead, e. g. `button.colors = new ControllerColors({...});`.

Reply:
I think ColorMapper is a good name.